src/base/sequence_checker.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include <optional>
-
+#include <utility>
```

Comment:
1. this should be in .cc file, not header, as the header doesn't need it
2. empty line between STL includes and library includes

Reply:
Done with required changes.